common/features/config/networks/static/reducer.ts

```diff
@@ -393,7 +394,8 @@ export const STATIC_NETWORKS_INITIAL_STATE: StaticNetworksState = {
       min: 0.183,
       max: 1.5,
       initial: 0.183
-    }
+    },
+    unsupportedTabs: [TAB.SWAP, TAB.ENS]
```

Comment:
swap should be supported still - no reason to disable it if people want to swap between various other coins/tokens :)

Reply:
@dternyak we are trying to avoid misunderstandings about the network, but it's true that this feature considers other networks